apps/site/pages/en/download/index.mdx

```diff
@@ -3,6 +3,8 @@ layout: download
 title: Download Node.js®
 ---
 
+<WindowsWSLMessage />
```

Comment:
This can be part of the Layout, right?

And can't AlertBox be used instead of a new component?

Reply:
`AlertBox` doesn't make a popup, I think you are thinking of `Modal`. If you select a community option for Download (rather than a Recommended), you'll see an alert box that says "Installation methods that involve community software are supported by the teams maintaining that software.", we can add one above for WSL.

It can say something like:

"These instructions are meant to be ran in a PowerShell terminal. If you are using [Windows Subsystem for Linux](link to ms docs), please refer to the Linux instructions"

Reply:
Sorry but I couldn't find the "Community Installation" method / option at nodejs website! 

can you please guide me through url or something so that I can get a rough idea about AlertBox

Reply:
@Vishal-K-988 If you select 'Windows' and 'Chocolatey' you should see the AlertBox component (it's the blue info box).